opengrok-web/src/main/webapp/js/utils-0.0.24.js

```diff
@@ -1716,6 +1716,11 @@ function pageReadyMast() {
 }
 
 function domReadyMenu(minisearch) {
+    if (getCookie('opengrok.suggester.enabled') === 'false') {
```

Comment:
maybe this should follow the common naming "scheme" used for pre-existing cookies ? i.e. `OpenGrokSuggester` ?

Reply:
I just used what came first to my mind. However, I found only one cookie in the whole project which has name `OpenGrokProject`.  Maybe there are some others?

For me `OpenGrokSuggester` is harder to read than `opengrok.suggester` and I think it would be worse if we use other modules (e.g. `OpenGrokIndexer`, ...).

But I agree that we should have a common naming. So `OpenGrokSuggester.enabled` it is? Or do you propose something different? :)

Reply:
There is another one, see the wiki. `OpenGrokSuggester.enabled` is good enough I guess.